core/services/versionchooser/utils/dockerhub.py

```diff
@@ -14,7 +14,7 @@
 
 def get_current_arch() -> str:
     """Maps paltform.machine() outputs to docker architectures"""
-    arch_map = {"x86_64": "amd64", "aarch64": "arm64", "armv7l": "arm32"}  # TODO: differentiate arm64v8/ arm32v7
+    arch_map = {"x86_64": "amd64", "aarch64": "arm", "armv7l": "arm"}  # TODO: differentiate arm64v8/ arm32v7
```

Comment:
Will this reject armv7 images running on armv6 ?

Reply:
armv6 will fail with "Unkown architecture" a few lines down